### ISSUE TITLE
FR: CSS Image support

### ISSUE THREAD

**Issue by Kayzels** [koreader/koreader#8391]
I'm not sure how hard this would be to implement. I've got a few epubs where images are declared in CSS rather than in the HTML. That is, there are decorative section break images mentioned in the CSS, for example:

`hr.section-break::after {
  content: url(some_image.jpg);
} `

In Calibre, the image shows. However in Koreader, instead of the image, you get a small diamond. It's not a big deal, but it would be cool if the image could be displayed. 

When it's a normal string declared the same way, it displays the string, so I'm not sure if it's just the way images are rendered?

**Comment by Frenzie** [comment]
A diamond is presumably the invalid glyph and/or glyph not found character.

**Comment by Kayzels** [comment]
Yeah, it's that character. Maybe because it is trying to process a URL as a string, rather than as a reference? 

I noticed in the CREngine pull request history that there was an update for background-image that supported using url() for the CSS value, but I'm not sure how it was done.

**Comment by Frenzie** [comment]
> Maybe because it is trying to process a URL as a string, rather than as a reference?

Something like that would be my guess, yes.

**Comment by poire-z** [comment]
Yes, unsupported for now in pseudo-elements' content:
https://github.com/koreader/crengine/blob/5f6c5da61428535b76277a75efd5ae059125efa6/crengine/src/lvstsheet.cpp#L1196-L1219
https://github.com/koreader/crengine/blob/5f6c5da61428535b76277a75efd5ae059125efa6/crengine/src/lvstsheet.cpp#L1282-L1284
and the diamond we explicitely show in this case:
https://github.com/koreader/crengine/blob/5f6c5da61428535b76277a75efd5ae059125efa6/crengine/src/lvstsheet.cpp#L1524-L1533
